package.json

```diff
@@ -34,7 +34,7 @@
     "can-diff": "^1.0.1",
     "can-dom-data": "^1.0.1",
     "can-dom-events": "^1.1.2",
-    "can-dom-mutate": "^1.0.3",
+    "can-dom-mutate": "^2.0.5",
```

Comment:
This should be bumped to `^1.3.11`. We will bump the `2.x` version on the `major` branch.

Reply:
@phillipskevin : bump to 1.x (instead of 2.x) as-requested.